packages/searchbox/tests/site/index.html

```diff
@@ -0,0 +1,378 @@
+		</script>
+	<![endif]-->
+</head>
+<body class="au-grid">
+	<noscript>
+		<p role="alert">This website needs JavaScript to work properly.</p>
+	</noscript>
+
+	<a class="test-link" href="../">&larr; back to the module index</a>
+
+	<h1>Test: searchbox</h1>
+
+	<div class="split-wrapper">
+
+		<div class="split">
+
+			<h2>Standard</h2>
+			<form role="search" class="au-search au-form-group">
+				<label for="standard" class="au-search__label">Enter a domain</label>
+				<input type="search" id="standard" name="standard" class="au-text-input" />
+				<div class="au-search__btn">
+					<button class="au-btn" type="submit"><span class="au-search__submit-btn-text">Search</span></button>
+				</div>
+			</form>
+
+
+			<h2>Responsive</h2>
+
+			<form role="search"  class="au-search au-search--responsive au-form-group">
+				<label for="responsive" class="au-search__label">Search</label>
+				<input type="search" id="responsive" name="responsive" class="au-text-input" />
+				<div class="au-search__btn">
+					<button class="au-btn"><span class="au-search__submit-btn-text">Search</span></button>
+				</div>
+			</form>
+
+			<h2>Icon only</h2>
+
+			<form role="search"  class="au-search au-search--icon au-form-group">
+				<label for="icon" class="au-search__label">Search</label>
+				<input type="search" name="icon" id="icon" class="au-text-input" />
+				<div class="au-search__btn">
+					<button class="au-btn"><span class="au-search__submit-btn-text">Search</span></button>
+				</div>
+			</form>
+
+			<br>
+			<h2>Different text and visible label</h2>
+
+			<label for="vis" class="au-label">Enter a domain</label>
+			<form role="search"  class="au-search au-form-group">
+				<input type="search" name="vis" id="vis" class="au-text-input" />
```

Comment:
Worth considering inclusion/exclusion or discussion of related attributes:

- `required`
- `autocomplete`
- `autocapitalize`
- `autocorrect`
- `spellcheck`

Reply:
we can add this on the docsite